src/views/Requests.svelte

```diff
@@ -43,6 +56,8 @@ function selectCreator(userId) {
     provider: null,
     size: null,
   })
+
+  filteredRequestsByMine()
```

Comment:
At the moment, we're only calling `selectCreator()` with `$me.id` as the given `userId`, so this works. However, nothing about the `selectCreator()` method is specific to _my_ requests.

I'd recommend changing the `RequestQuickFilter` to call a new `showMyRequests()` method, and have that method call `selectCreator($me.id)` and `filteredRequestsByMine()`.

Reply:
I like that change, I've got some other refactor-type suggestions around filtering I'd like to discuss and group all the related changes into some commits outside of GA.